### ISSUE TITLE
bug: class do not apply to both inputs

### ISSUE THREAD

**Issue by ashwin-nath-m** [paul-thebaud/v-phone-input#37]
## Expected Behavior
Margin should be applied to both country select and phone input

## Actual Behavior
margin only applied to phone input

## Steps to Reproduce the Problem

```
<v-phone-input v-model="mobile" defaultCountry="US" guessCountry="true" variant="underlined"
                            class="mt-3"></v-phone-input>
```

## Specifications

  - OS: OSX
  - Browser: Firefox
  - Browser version: 127
  - Package version: 4.2.1
  - Vuetify version: 3.6.10

**Comment by paul-thebaud** [comment]
Hello, I can confirm there is a bug here. When passing `class` prop, it applies to both wrapper div and phone input, where it should only apply to wrapper div. I'll try to fix this soon.

If you need to pass attributes to both inputs, prefer using `country-props` and `phone-props` properties.

**Comment by paul-thebaud** [comment]
@ashwin-nath-m this is fixed in v4.2.2. [Documentation about attrs inheritance](https://github.com/paul-thebaud/v-phone-input?tab=readme-ov-file#props-inheritance) is now in the readme.
